tutorials/review.adoc

```diff
@@ -0,0 +1,881 @@
+<6> The committee responsible for the document
+<7> The author’s name. You can add co-authors by appending the attribute with a number:  `_2`, `_3`, and so on. 
+<8> When you add an address that contains multiple lines, end each line except for the last with `+ \`. Alternatively, you can enter each line into its own attribute, such as `:street:`, `:postcode:`, `:city:`, `:country:`. 
+
+[[exercise-2-1]]
+include::exercises.adoc[tag=exercise-2-1]
+
+Let’s add some sections to our document in the next lesson.
+>> Start next lesson 
+
+=== Sections
+
+Sections define hierarchy levels in your document and are equivalent to chapters. The document title is the highest section and is created by prepending a `=` sign in front of the heading. If you want to go one level deeper in the hierarchy, add another `=`. 
+Here’s an example: 
+
+[source, AsciiDoc]
+----
+= Document title
+== Section 1
+=== Section 1.1
+==== Section 1.1.1
+===== Section 1.1.1.1
+== Section 2
+…
+----
+
+Metanorma provides a document model that determines what sections need to appear in a document. Let’s have a look at the standard document model (standoc) to understand what kinds of sections appear in a typical Metanorma document.
+
+.A UML representation of all section classes in Metanorma
+image::https://raw.githubusercontent.com/metanorma/metanorma-model-standoc/master/images/StandardDoc_Sections.png[A UML representation of all section classes in Metanorma]
+
+A typical Metanorma document can contain the following sections:
+
+* Abstract
+* Foreword
+* Introduction
+* Acknowledgments
+* Scope
+* Terminology 
+* Definitions
+* Clauses 
+* Bibliography
+* Annex
+
+NOTE: Each organization is based on the standard document model but they can omit sections or make them mandatory, if they choose to. For example, only NIST uses the acknowledgments section, whereas other SDOs do not require it. Check the https://www.metanorma.org/flavors/[flavor documentation] for more details on how your SDO uses Metanorma. 
+
+To successfully validate a document, the compiler needs to know what sections are in the document. Most of the time, the section headings match sections defined in the document model. However, not all sections need an identifier, for example, your ordinary content sections as they fall under the category of `clause section`, and are not further standardized
+
+If you want to use a different title or create a document in a language other than English, you need to add metadata in square brackets to ensure that sections are recognized. Plain AsciiDoc already provides some https://docs.asciidoctor.org/asciidoc/latest/sections/section-ref/#section-styles[section metadata] that Metanorma uses for document validation: 
+
+* `[abstract]`
+* `[acknowledgments]`
+* `[appendix]`
+* `[bibliography]`
+* `[index]`
+* `[preface]`
+
+There are also some Metanorma-specific section identifiers:
+
+* `[terms and definitions]`
+* *Any more?* 
```

Comment:
Yeah, "Scope" (applicable to ISO), "Symbols and abbreviated terms", "Introduction", "Foreword". Again, these need to be given as `heading=`